### DISCUSSION TITLE
How can I use prisma 2 args on graphql resolver

### DISCUSSION THREAD

**Discussion by cancng** [prisma/prisma#6289] [Q&A]
Hi. I want to use prisma 2 arguments on my graphql query or mutations. It is possible with typegraphql-prisma but our project is now pure apollo-server and prisma 2 ORM.

For example prisma 2 args here related to generated prisma schema

![prisma 2 args](https://cdn.discordapp.com/attachments/747053914477559828/825758229459107880/unknown.png)

**Comment by ryands17** [comment]
@cancng 👋 

If you're not using a generator like `typegraphql-prisma`, then you would need to create the input types manually in your `schema.graphql` with the inputs that Prisma takes. You can have a look at [this](https://github.com/prisma/prisma-examples/blob/latest/javascript/graphql-sdl-first/src/schema.js) example on how to create input types.

**Reply by cancng** [reply]
Unfortunately we didnt use `type-graphql` this time. Could you fix `this` link? I want to look example. Thanks 👋

**Reply by ryands17** [reply]
Done :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
